### PR TITLE
[ Header ] Avatar Image , Github Logged in Button

### PR DESCRIPTION
**Pull request by darshkpatel**
added avatar from GitHub profile, more generic placeholder image and logged in button ( static since dashboard can only be seen once logged in via GitHub )

### CODE REVIEW

File: src/containers/TheHeader.js [hunk 1]
```diff
@@ -79,6 +79,11 @@ const TheHeader = () => {
               <i className="cib-discord"> </i> Discord Logged In
             </button>
           )}
+          <div style={{paddingLeft: 10}}>
```
Comment (sauravhiremath) [inline src/containers/TheHeader.js:82]: Could you check if the `paddingLeft` is optically aligned even when the page is checked for responsive sizes?

**Review by sauravhiremath — CHANGES_REQUESTED** [review]
Reviewed it. LGTM, added a comment. Should be gtg after that

> I think we can start adding the screenshots of the UI change in the PRs too. Coz most of our changes would not be visible due to login filter

File: src/containers/TheHeader.js (hunk 1 above)
Comment (darshkpatel) [reply src/containers/TheHeader.js:82]: yes it is

**Review by sauravhiremath — APPROVED** [review]

File: src/containers/TheHeader.js (hunk 1 above)
Comment (darshkpatel) [reply src/containers/TheHeader.js:82]: these buttons are not visible on mobile

**Pull request merged**